src/PreCICE.jl

```diff
@@ -508,6 +532,39 @@ function setMeshEdge(meshName::String, firstVertexID::Integer, secondVertexID::I
     )
 end
 
+@doc """
+
+    setMeshEdges(meshName::String, vertices::AbstractArray{Cint})
+
+Create multiple mesh edges
+
+# Arguments
+- `meshName::String`: Name of the mesh to add the edge to.
+- `vertices::AbstractArray{Cint}`: An array holding the vertex IDs of the edges.
+                                   It has the shape [N x 2] where N = number of edges.
+
+# Examples
+Set mesh edges for a problem with 4 mesh vertices in the form of a square with both diagonals which are fully interconnected.
+```julia-repl
+julia> vertices = [1 2; 1 3; 1 4; 2 3; 2 4; 3 4]
+julia> vertices.shape
+(6,2)
+julia> setMeshEdges("MeshOne", vertices)
+```
+"""
+function setMeshEdges(meshName::String, vertices::AbstractArray{Cint})
+    _size, dimensions = size(vertices)
+    @assert dimensions == 2 "Dimensions of vector data in write_vector_data does not match with dimensions in problem definition. Provided dimensions: $dimensions, expected dimensions: $(getDimensions())"
```

Comment:
Assertion message seems like a left over from copy pasting. Lets write a more sensible message here. For example, *Vertices pairs need to be provided while setting mesh edges. Please check the shape of the vertices array*.